R/get_nhdplushr.R

```diff
@@ -59,7 +60,7 @@ download_nhdplushr <- function(nhd_dir, hu_list, download_files = TRUE) {
       if(download_files & !dir.exists(gsub(".zip", ".gdb", out_file)) &
          !dir.exists(file.path(dirname(out_file), paste0(hu04, ".gdb")))) {
```

Comment:
this will be a useful place to highlight a HUC spatial discovery tool

Reply:
For sure. I'll open an issue as a todo.